.github/workflows/main.yml

```diff
@@ -46,11 +46,22 @@ jobs:
           conda install --file requirements/test.txt
           pip install .
 
+      # redownload diffpy.utils pre 3.2.0 release, use https since public repo
```

Comment:
Please make sure there is an issue to remove this when diffpy.utils is released on conda-forge.  I would rather not do this here tbh, but if it is needed for now it is ok, but let's make sure we remove it later.

What is left on diffpy.utils to get it released?  Can we just do that and not have this here?  it is just making more work in the future.....